preprocess/pre_process.py

```diff
@@ -10,8 +10,15 @@ def clean_data(location, method):
     df_raw = pandas.read_csv(location)
 
     """
-    add logic to clean data
+  import pandas as pd
+  import numpy as np
+
+  df_raw=pd.read_csv('Source/test1.csv')
+  df1=df_raw.replace('[^0-9]','',regex=True).astype(float)
+  df2=df1.fillna(df1.mean())
```

Comment:
pre process function takes two arguments(location , method)

Use method to determine the type of cleaning.

ex:
If method == "Drop":
    Do dropna()
elif method == "Prev":
    Use replace null by previous value.
elif method == "avg":
    add avg